src/relax/ir/dataflow_matcher.cc

```diff
@@ -663,11 +758,51 @@ static std::optional<MatchState> TryMatch(const PNode& p, const RNode& r,
   return new_match;
 }
 
+static std::optional<MatchState> TryValidate(
+    const MatchState& current_match,
+    const std::unordered_map<const DFPatternNode*, PNode>& pattern2node,
+    const std::vector<DFConstraint>& validation_constraints,
+
```

Comment:
Remove?

Reply:
Thank you, and the empty line is now removed.